### DISCUSSION TITLE
starting codespace is too slowly under the Chinese Network

### DISCUSSION THREAD

**Discussion by howel52** [community/community#3421] [Codespaces]
I try to create a codespace, but requesting static resources of the codespace page very slowly.  Maybe one of the reason is that my network provider is a Chinese company. 

![image](https://user-images.githubusercontent.com/9854818/108959492-6de4d600-76af-11eb-9570-e6d05d1a077e.png)

Has any plan to improve the CDN service in China？

**Comment by PranamBhat** [comment]
I think @evi-liu @candyho can help you on the same.
